app/controllers/admin/events_controller.rb

```diff
@@ -9,11 +9,21 @@ def new
     def create
       @event = Event.new(event_params)
 
-      event.save ? default_redirect : render(:new)
+      if event.save
+        flash[:success] = t('events.notifications.create.success')
```

Comment:
@denysmedynskyi i think we need DRY.
Implement some method like `flash_success(:events)`